apps/campus/management/commands/import_address_data.py

```diff
@@ -0,0 +1,35 @@
+import csv
+
+from django.core.management.base import BaseCommand, CommandError
+from django.conf import settings
+from django.db import connection, transaction
```

Comment:
Some of these imports don't appear to be in use--consider removing if they're not necessary